syntheseus/cli/search.py

```diff
@@ -187,6 +189,7 @@ def run_from_config(config: SearchConfig) -> Path:
                 "limit_reaction_model_calls",
                 "limit_iterations",
                 "prevent_repeat_mol_in_trees",
+                "limit_graph_nodes",
```

Comment:
Just keeping the same order as above 😄
```suggestion
                "limit_graph_nodes",
                "prevent_repeat_mol_in_trees",
```

(This "suggestion" feature of GitHub is making me really nitpicky, because with it I assume the cost of applying the change for the creator of the PR is effectively zero, leading me to propose changes I wouldn't bother with otherwise...)